CHANGELOG.md

```diff
@@ -1,22 +1,32 @@
 # Change Log
 
 ## v1.3.0
+
+- [New] Add `direction` option (prop) to `withDirection` HOC, This to allow the overwrite of the direction when required (for phone numbers for example that are always LTR)
```

Comment:
v1.3.0 is already released; if you're going to add changelog items, please add in an "unreleased" section

Reply:
Reverted the changes to the file, I'll update this once the PR is approved and bump the version on this package.

Reply:
note; when merged, in a commit directly made to master.